clustermesh/clustermesh.go

```diff
@@ -466,8 +469,14 @@ type Parameters struct {
 	ConfigOverwrites     []string
 	Retries              int
 	HelmValuesSecretName string
+	Output               string
 }
 
+const (
+	OutputJSON    = "json"
+	OutputSummary = "summary"
+)
```

Comment:
Let's define these in the `status` package instead, so we can use them here and in `internal/cli/cmd`.